### DISCUSSION TITLE
Lesson 6 "Nothing to compile" error

### DISCUSSION THREAD

**Discussion by knowbuddycrypto** [Cyfrin/foundry-full-course-cu#2140] [Q&A]
I keep getting "nothing to compile" error. I'm pretty sure I'm in the right folder, and it's the first time I'm compiling, so I don't think it's because I'm not compiling something new. Any help?
![Capture](https://github.com/user-attachments/assets/35f7d2c1-a1a5-427e-87e3-37290d6120e9)

**Comment by EngrPips** [comment]
Hello @knowbuddycrypto, I want to mention that I have an eye on this, but it seems difficult to catch, lol.

**Reply by knowbuddycrypto** [reply]
Thanks for the reply! Hopefully it's something easy to remedy
